python/paddle/fluid/transpiler/distribute_transpiler.py

```diff
@@ -225,6 +225,7 @@ def transpile(self,
                 inputs={"X": splited_vars},
                 outputs={},
                 attrs={
+                    "sync_mode": self.sync_mode,  # convert to int
```

Comment:
Please add a TODO comment, and we need to make it as `async_send` to improve performance.

Reply:
Done